### DISCUSSION TITLE
Filenames not displayed in window title bar, on v3.49.0

### DISCUSSION THREAD

**Discussion by Nepenthy** [mifi/lossless-cut#1435] [Support]
Hello, longtime user of the app, love it. Ive just noticed after updating, that filenames no longer show in the window title bar. I am running multiple instances, 10 to be exact. This has never been an issue in the past, outside of adding a command line switch to allow it. When editing that many files simultaneously, it was a big help to be able to see the filename in the window im working on. Did i miss a setting or something? I cant imagine a reason for someone to request removal of that function. Running on Win 11.

**Comment by mifi** [comment]
Ahh, that's a bug. I added support for showing cut progress in the title, but that has overwritten the file name. will fix. thanks for reporting
